runtime/src/main/resources/messages.properties

```diff
@@ -14,5 +14,9 @@
 # limitations under the License.
 #
 
-io.syndesis.credential.RelativeUri.message = Given URI value must be relative
-io.syndesis.model.validation.UniqueProperty.message = Value `${validatedValue[value]}` for property `{value}` is not unique
+io.syndesis.credential.RelativeUri.message          = Given URI value must be relative
```

Comment:
Not deep in bean validation, but can the value of the URI included in the error message ? Or is this done by default ? The more context, the better ...